server/src/users/userController.js

```diff
@@ -74,41 +78,37 @@ const findUser = (req, res) => {
 
 //Adduser if not created, otherwise will return user info
 const addUser = (req, res) => {
-  User.count({
-    where: {
-      name: req.body.username
-    }
-  })
-  //find if user is currently in database, 
-  .then(count => {
-    //if in database, throw err
-    if (count !== 0) {
-      console.log('User already exists');
-      res.end()
-    } else {
-      User.create({
-        name: req.body.username,
-        accessToken: req.body.token,
-        slackUserId: '1111', /*some slack_id queried using slack token*/
-        teamId: '2222' /*some team_id queried using slack token*/
-      })
-      .then(user => {
-        console.log('Created new user!');
-        res.end();
-      })
-      .catch(err => {
-        console.log('Error creating user...');
-        done(err)
-      }) 
-    }   
-    //else create user
-    //when a new user is added, need to generate a hash
-  })
-  .catch(err => {
-    console.log('Error: ', err);
-    done(err);
-  })
-    //else create the user
+  let users = req.body.users;
+  let teamId = req.body.teamId;
+  let accessToken = null;
+
```

Comment:
Leave a comment here to let future devs know that this function expects the users to be sent in an array, even if it's a single user